openssl-dynamic/src/main/c/ssl.c

```diff
@@ -1489,7 +1492,7 @@ TCN_IMPLEMENT_CALL(jlong, SSL, makeNetworkBIO)(TCN_STDARGS,
         return 0;
     }
 
-    if (BIO_new_bio_pair(&internal_bio, 0, &network_bio, 0) != 1) {
+    if (BIO_new_bio_pair(&internal_bio, max_bio_size, &network_bio, max_bio_size) != 1) {
```

Comment:
There is an implicit cast of signed -> unsigned here which may cause warnings, which may be treated as errors depending on environment, on some compilers. Can you be explicit about the type conversion?